tests/gnmi/helper.py

```diff
@@ -63,6 +90,10 @@ def gnmi_capabilities(duthost, localhost):
     cmd += "-logtostderr -client_crt ./gnmiclient.crt -client_key ./gnmiclient.key -ca_crt ./gnmiCA.pem -capabilities"
     output = localhost.shell(cmd, module_ignore_errors=True)
     if output['stderr']:
+        # Dump gnmi log
```

Comment:
># Dump gnmi log [](http://example.com/codeflow?start=8&length=15)

Repeated code, could you extract a method?

Reply:
Fixed